src/main/java/io/vlingo/xoom/starter/task/projectgeneration/archetype/Archetype.java

```diff
@@ -28,9 +30,10 @@
     private final String artifactId;
     private final String groupId;
     private final String version;
-
     private final List<ArchetypeOption> archetypeOptions = new ArrayList<>();
 
+    private static final String POM_PATH_PATTERN = "%s%s%spom.xml";
```

Comment:
Extracting format strings is usually considered a [bad](https://errorprone.info/bugpattern/InlineFormatString) practice.